core/arch/arm/plat-imx/link.mk

```diff
@@ -1,3 +1,15 @@
 include core/arch/arm/kernel/link.mk
 
 all: $(link-out-dir)/tee-raw.bin
+
+cleanfiles += $(link-out-dir)/entry_point_address.txt
+$(link-out-dir)/entry_point_address.txt: $(link-out-dir)/tee.elf
+	@$(cmd-echo-silent) '  GEN     $@'
+	$(q)$(READELFcore) -h $(link-out-dir)/tee.elf | grep "Entry point address" | awk '{print $$4}' > $@
```

Comment:
`grep "Entry point address" | awk '{print $$4}'` can also be written as:
`awk /Entry point address/ '{print $$4}'`

Reply:
Ok, fixed!